# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/clifford/mysqlcc/qmakespec.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./configure.in.orig	2003-01-26 19:28:40.000000000 +0100
+++ ./configure.in	2003-01-26 19:28:49.000000000 +0100
@@ -24,7 +24,7 @@
 AC_SUBST(MYSQLCC_HEADERS)
 AC_SUBST(MYSQLCC_SOURCES)
 AC_OUTPUT(mysqlcc.pro)
-QMAKESPEC=$QTDIR/mkspecs/$host_os
+QMAKESPEC=$QTDIR/mkspecs/linux-g++
 echo "Creating Makefile with qmake"
 QMAKESPEC=$QMAKESPEC QTDIR=$QTDIR $QTDIR/bin/qmake mysqlcc.pro
 if test ! $? = 0 ; then
